google-cloud-graalvm-samples/quarkus-pubsub-sample/src/main/docker/Dockerfile.fast-jar

```diff
@@ -0,0 +1,57 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
```

Comment:
Was this file auto-generated by something?

Reply:
Yup - this code was generated from the Quarkus equivalent of Initialzr.